<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChiTieu</class>
 <widget class="QDialog" name="ChiTieu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover:!pressed
{
  	border: 4px solid white;
}


QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    
	image: url(:/Images/Pictures/Arrows-Down.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
</string>
  </property>
  <widget class="QLabel" name="label_DenDanhMucHienTai">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>161</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Danh mục hiện tại&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SoTien">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);

border-radius: 10px;
border: 2px solid #e1eff6;</string>
   </property>
   <property name="placeholderText">
    <string>...đ</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_TuTaiKhoan">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border: 2px solid #e1eff6;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 14pt &quot;Segoe UI&quot;;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;Segoe UI&quot;;




</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>9</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_MoTa">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;
border-radius: 10px;
border: 2px solid #e1eff6;</string>
   </property>
   <property name="placeholderText">
    <string>Tên mặt hàng, thông tin,..</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ThemChiTieu">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;THÊM CHI TIÊU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>65</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_TuTaiKhoan">
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Từ tài khoản:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_DenDanhMuc">
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Đến danh mục:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_SoTien">
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Số tiền:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_p4_TenTaiKhoan">
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Ngày chi tiêu:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_p4_MoTa">
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Mô tả:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 97, 205, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn_OK">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(255, 255, 255);
background-color: rgb(255, 106, 108);
font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Huy">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(255, 255, 255);
background-color: rgb(255, 106, 108);
font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Huỷ</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label_DenDanhMucHienTai</zorder>
  <zorder>lineEdit_SoTien</zorder>
  <zorder>comboBox_TuTaiKhoan</zorder>
  <zorder>dateEdit</zorder>
  <zorder>textEdit_MoTa</zorder>
  <zorder>label_ThemChiTieu</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
